File: proto/proto.py
class Food():

    def __init__(self, type):
        self.type = type
        self.something = 15

    def eat(self):
        print('you ate the ' + self.type)


waffle = Food('waffle')


print(waffle.type)
print(waffle.something)

Food.type = 'asdasd'
Food.something = 'asdasd'
Food.x = 45

print(waffle.type)
print(waffle.something)
print(waffle.x)

File: proto/proto.js
var Food = function(type) {
   this.type = type;
   this.something = 15;
};

Food.prototype.eat = function(){
   console.log('you ate the ' + this.type);
};
//
var waffle = new Food('waffle');

console.log(waffle.something);

Food.something = 'asdasd'

console.log(waffle.something);

var cookie = new Food('cookie');

console.log(Food);
console.log(cookie.something);
